src/main/java/com/codecool/virtualpub/ui/Display.java

```diff
@@ -24,7 +24,7 @@ public void displayMessage(String message) {
     public void displayScript(BartenderScript script, int index) {
         try{
             System.out.println("\n" + "Bartender: " + script.getScripts().get(index) + "\n");
-        } catch (Exception e) {
+        } catch (IndexOutOfBoundsException e) {
```

Comment:
Why can we get an indexOutOfBoundException? Can't we ensure that the index will always be inside the range?